### PR TITLE
i18n: Link & Breadcrumbs

### PR DESCRIPTION
**Pull request by nhsz**
This PR adds the `locale` prop from router to `Link` and `Breadcrumbs` components for localized navigation between internal pages

### CODE REVIEW

File: src/components/Breadcrumbs/index.tsx [hunk 1]
```diff
@@ -39,13 +40,10 @@ const Breadcrumbs: React.FC<IProps> = ({
 }) => {
   // TODO: update when i18n is set up
   // const { t } = useTranslation()
-  // const { language } = useI18next()
-  const language = "en"
+  const { locale } = useRouter()
 
-  // TODO: update when i18n is set up
-  // const hasHome = originalSlug.includes(`/${language}/`)
```
Comment (pettinarip) [inline src/components/Breadcrumbs/index.tsx:46]: Why this TODO was deleted? or should we remove the `hasHome` flag?

**Review by pettinarip — COMMENTED** [review]
Nice.

~~There is [one other case](https://github.com/ethereum/ethereum-org-fork/blob/frontmatter-typing/src/components/TranslationBanner.tsx#L106) where we need to force the Link to navigate to a specific locale. Before, we had a prop called `language` that covered this purpose. Wondering if we should add it back or think of a workaround.~~

edit: nvm, saw that you are covering this case in #55

File: src/components/Breadcrumbs/index.tsx (hunk 1 above)
Comment (nhsz) [reply src/components/Breadcrumbs/index.tsx:46]: Oops, wrong deletion, will fix

Comment (nhsz) [reply src/components/Breadcrumbs/index.tsx:46]: Fixed on last commit

**Review by pettinarip — APPROVED** [review]

**Pull request merged**